### PR TITLE
[DOC] Adding recommendations into multi-echo.rst

### PR DESCRIPTION
**Pull request by handwerkerd**
Related to discussion in  #340 and #336 

Changes proposed in this pull request:
- Two paragraphs are added to multi-echo.rst to discuss "Recommendations on multi-echo use for someone planning a new study" This addition is in response to several requests at OHBM2019. Several people should read this to make sure they agree with my recommendations!

- I also added stubs to this file where we can add information on "Available multi-echo fMRI sequences for multiple vendors" and "Multi-echo preprocessing software." I'm working on the first one, but I figured it was good to make a public place holder in case anyone sees it and asks to contribute. For the second one, the necessity of the other preprocessing pipelines is also mentioned in usage.rst, but it's the type of information that might be useful to include in two different locations (with more expansive text in one of those locations)

### CODE REVIEW

File: docs/multi-echo.rst [hunk 1]
```diff
@@ -54,6 +54,30 @@ We can use this information to denoise the optimally combined time series.
 .. _processing pipeline details: https://tedana.readthedocs.io/en/latest/approach.html#optimal-combination
 .. _Pruim et al. (2015): https://www.sciencedirect.com/science/article/pii/S1053811915001822
 
+Recommendations on multi-echo use for someone planning a new study
+------------------------------------------------------------------
+Multi-echo fMRI acquisition sequences and analysis methods are rapidly maturing. Someone who has access
+to a multi-echo fMRI sequence should seriously consider using it. Multiple studies have shown that a
+weighted average of the echoes to optimize T2* weighting, sometimes called "optimally combined,"
+gives a reliable, modest boost in data quality. The optimal combination of echoes can currently be
+calculated in several software packages including AFNI, FMRI_PREP, and tedana. If no other acquisition
```
Comment (jbteves) [inline docs/multi-echo.rst]:
```suggestion
calculated in several software packages including AFNI, fMRIPrep, and tedana. If no other acquisition
```
Nitpicky but none of the capitalizations on their [readthedocs](https://fmriprep.readthedocs.io/en/stable/index.html) have an underscore.

File: docs/multi-echo.rst [hunk 2]
```diff
@@ -118,6 +142,22 @@ Videos
...
+********************************************************
+
+Information on multi-echo sequences from Siemens, GE, and Phillips will be added here.
+
+Multi-echo preprocessing software
+*********************************
+
+tedana requires data that has already been preprocessed for head motion, alignment, etc.
+More details on software packages that include preprocessing options specifically for multi-echo
+fMRI data, such as AFNI and FMRI_PREP will be added here.
```
Comment (jbteves) [inline docs/multi-echo.rst]: Same as above

File: docs/multi-echo.rst [hunk 3]
```diff
@@ -54,6 +54,30 @@ We can use this information to denoise the optimally combined time series.
...
+it is likely that more powerful approaches to multi-echo denoising will sufficiently mature and add
+even more value to a data set.
+
+Other multi-echo denoising methods, such as MEICA, the predecessor to tedana, have shown the potential for
+much greater data quality improvements, as well as the ability to more accurately separate visually similar
+signal vs noise, such as scanner based drifts vs slow changes in BOLD signal. These more powerful methods are
+still being improved, and the algorithms are still changing. Users need to have the time and knowledge to look
+at the denoising output from every run to make sure denoising worked as intended. If someone wants a push-button
+way to use multi-echo data to improve data quality, that doesn't require as deep an inspection of every output,
+stick with using the weighted average. The developers of tedana look forward to when tedana and other methods
```
Comment (jbteves) [inline docs/multi-echo.rst]: I feel that we should link here to how to run *just* optimal combination for ease of reference.

Comment (handwerkerd) [reply docs/multi-echo.rst]: @jbteves, What specifically would you recommend? A link to: https://tedana.readthedocs.io/en/latest/usage.html#run-t2smap

Comment (jbteves) [reply docs/multi-echo.rst]: Yeah, sorry, I should have been more clear; I think just dropping that link in would be perfect!

File: docs/multi-echo.rst [hunk 4]
```diff
@@ -54,6 +54,31 @@ We can use this information to denoise the optimally combined time series.
...
 .. _Pruim et al. (2015): https://www.sciencedirect.com/science/article/pii/S1053811915001822
 
+Recommendations on multi-echo use for someone planning a new study
+------------------------------------------------------------------
+Multi-echo fMRI acquisition sequences and analysis methods are rapidly maturing. Someone who has access
+to a multi-echo fMRI sequence should seriously consider using it. Multiple studies have shown that a
+weighted average of the echoes to optimize T2* weighting, sometimes called "optimally combined,"
+gives a reliable, modest boost in data quality. The optimal combination of echoes can currently be
+calculated in several software packages including AFNI, fMRIPrep, and tedana. In tedana, the weighted
+average can be calculated with https://tedana.readthedocs.io/en/latest/usage.html#run-t2smap If no other
```
Comment (jbteves) [inline docs/multi-echo.rst]:
```suggestion
average can be calculated with t2smap_ If no other
```
Paired with a below suggestion would make the link in-line, which would improve readability slightly.

Comment (jbteves) [reply docs/multi-echo.rst]: At the end, with the other links at the bottom of the file, add
` .. _t2smap: https://tedana.readthedocs.io/en/latest/usage.html#run-t2smap

**Comment by jbteves** [comment]
@handwerkerd you have to also drop the link at the bottom of the page or it won't know which page to route it to. See my comment [here](https://github.com/ME-ICA/tedana/pull/341#discussion_r296294416) which unhelpfully wasn't a suggested change; the bottom of the page wasn't something you changed, so I couldn't suggest it in the review. Once that's done this is ready to merge IMO.

**Pull request merged**